build/hydra.nix

```diff
@@ -93,6 +93,9 @@ in
 
     max_concurrent_evals = 1
 
+    # increase the number of active compress slots (CPU is 48*2 on mimas)
+    max_local_worker_threads = 144
```

Comment:
Now we had a longer period of "Copying Results" hovering roughly around 130, and PSI kept still very low.  So I believe this certainly still isn't a too high value.  We'll see over time.  So far it's been relatively rare to get so high anyway.